rewrite-gradle/src/test/java/org/openrewrite/gradle/java/UseJavaxNullabilityAnnotationsTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.gradle.java;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.config.Environment;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.gradle.Assertions.buildGradle;
+import static org.openrewrite.gradle.Assertions.settingsGradle;
+import static org.openrewrite.java.Assertions.*;
+
+/**
+ * <Beschreibung>
+ * <br>
+ *
+ * @author askrock
+ */
```

Comment:
I suggest we remove the Javadoc above.

Reply:
I did not mean to commit that, that's how it was configured on my other device, sorry about that. 🙈